### PR TITLE
Enhancement: Delete total header from admin API endpoints

### PR DESCRIPTION
**Pull request by chumaknadya**
Remove Total header from endpoints:
api/v2/admin/operations
api/v2/admin/orders
api/v2/admin/trades
api/v2/market/orders
api/v2/market/trades

### CODE REVIEW

File: app/api/v2/helpers.rb [hunk 1]
```diff
@@ -83,6 +83,39 @@ def format_ticker(ticker)
         { at: ticker[:at],
           ticker: formatted_ticker }
       end
+
+      def paginate(collection, options = {})
```
Comment (mnaichuk) [inline app/api/v2/helpers.rb]: use can set here default for true

File: app/api/v2/helpers.rb [hunk 2]
```diff
@@ -83,6 +83,39 @@ def format_ticker(ticker)
...
+        pages.each do |k, v|
+          old_params = Rack::Utils.parse_nested_query(request.query_string)
+          new_params = old_params.merge('page' => v)
+          links << %(<#{url}?#{new_params.to_param}>; rel="#{k}")
+        end
+
+        total_header    = ApiPagination.config.total_header
+        per_page_header = ApiPagination.config.per_page_header
+        page_header     = ApiPagination.config.page_header
+        include_total   = options[:include_total].nil? ? ApiPagination.config.include_total : options[:include_total]
```
Comment (mnaichuk) [inline app/api/v2/helpers.rb]: and put here only options check if we pass false it will not include if it use true it will incude

**Review by mnaichuk — CHANGES_REQUESTED** [review]
For adjustments, blockchains, currencies, makrets, members, trading_fees and wallets we can keep.
We need for tables with a large amount of data like all operations, order and trades.

**Review by mnaichuk — APPROVED** [review]

**Review by mod — CHANGES_REQUESTED** [review]
If I understand you copied and modified helper paginate from the gem ?
Do we need the gem at all ?
How will be the usage with just Kaminari

This code doesn't feel right

File: app/api/v2/helpers.rb [hunk 3]
```diff
@@ -83,6 +83,17 @@ def format_ticker(ticker)
         { at: ticker[:at],
           ticker: formatted_ticker }
       end
+
+      def paginate(collection, include_total = true)
+        per_page = params[:limit].present? ? params[:limit].to_i : Kaminari.config.default_per_page
```
Comment (mnaichuk) [inline app/api/v2/helpers.rb]: par_page = params[:limit] || Kaminari.config.default_per_page

**Review by mnaichuk — CHANGES_REQUESTED** [review]

**Pull request merged**